libcxx/include/deque

```diff
@@ -1202,10 +1203,10 @@ private:
   template <class _Iterator>
   _LIBCPP_HIDE_FROM_ABI iterator __insert_with_size(const_iterator __p, _Iterator __f, size_type __n);
 
-  template <class _BiIter, class _Sentinel>
+  template <class _IterOpsPolicy, class _BiIter, class _Sentinel>
```

Comment:
```suggestion
  template <class _AlgPolicy, class _BiIter, class _Sentinel>
```
I think the name is close enough to being correct here for it to make sense to keep the original naming scheme. Throughout.